packages/rest/test/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -89,6 +89,29 @@ describe('RestServer.getApiSpec()', () => {
     });
   });
 
+  it('ignores routes marked as "x-internal" via app.route(route)', () => {
+    function greet() {}
+    function meet() {}
+    server.route(
+      new Route(
+        'get',
+        '/greet',
+        {'x-internal': true, responses: {}, spec: {}},
+        greet,
+      ),
+    );
+    server.route(new Route('get', '/meet', {responses: {}, spec: {}}, meet));
+    const spec = server.getApiSpec();
+    expect(spec.paths).to.eql({
+      '/meet': {
```

Comment:
See my comment above. In this test, it should be enough to verify that when the app has only a single route that's internal, then the spec object has no paths.

```ts
expect(spec.paths).to.eql({});
```

Reply:
Same as above, verify for no side-effects.